POS/Integration/06_CreateStore.md

```diff
@@ -170,26 +184,25 @@ The store wizard has been started, see [Start the store wizard](#start-the-store
 
 ![Wizard 05](../../Assets/Screenshots/POS/Management/Stores/StoreWizard/SW05.png "[Store Wizard]")
 
-1. Select the warehouse you want to take the stock from in the *Warehouse* drop-down list.
+1. Click the *Warehouse* drop-down list and select the warehouse you want to take the stock from.
 
 2. Activate the *Automatically assign product to warehouse when activated in store* toggle to automate the assignment process. It is recommended to activate this option.
 
-    > [Info] A product must have stock in a warehouse in order to be sold from that warehouse. Products only have stock in a warehouse if a shelf is defined for them. By default, a shelf is not defined for products. Therefore, each product that is to be sold in the POS system must first get a shelf. The products get a shelf the first time the corresponding channels offer is activated.    
+    > [Info] A product must have a stock level in a warehouse in order to be sold from that warehouse. Products only have stock level in a warehouse if a shelf is defined for them. By default, a shelf is not defined for products. Therefore, each product that is to be sold in the POS system must first get a shelf. The products get a shelf the first time the corresponding sales channels offer is activated. 
+
     This automation only works if you have enabled the storage of multiple items per shelf when creating the warehouse. For detailed information, see [Configure the warehouse for POS](./01_ConfigureWarehouse.md).
 
 3. Enter a shelf number for the products in the *Shelf* field. The shelf number can be any arbitrary number.
 
-    > [Info] A product must have stock in a warehouse in order to be sold from that warehouse. Products only have stock in a warehouse if a shelf is defined for them. By default, a shelf is not defined for products. Therefore, each product that is to be sold in the POS system must first get a shelf.
-
 4. Activate the *Configure stock withdrawal matrix* toggle to automate the configuration of the stock withdrawal matrix. It is recommended to activate this option.
 
 5. Click the [CONTINUE] button in the bottom right corner of the window.   
     The *Stock allocation* wizard window is displayed when the warehouse assignment is saved.
 
 
-### Assign a stock source
+## Assign a stock source
 
-In this step you assign the stock source to manage the store stock. The stock source specifies how the stock available in the POS system is calculated. In most cases, the stock source and the warehouse are identical, but they can also differ.
+In this step you assign the stock source to manage the store stock level. The stock source specifies how the stock level in the POS system is calculated. In most cases, the stock source and the warehouse are identical, but they can also differ.
```

Comment:
The stock source (sollte "allocation" sein) specified ow the stock VALUE in the POS is calculated...

Dann beziehen sich auf die Stock allocation matrix, auch wenn sie es anders nennen (stock source).